salt/personalised-covers/config/srv-personalised-covers-config-config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+	{% if pillar.elife.env in ['ci', 'dev']  %}
+	'debug' => true,
+	{% else %}
+	'debug' => false,
+	{% endif %}
+	'aws_region' => "{{ pillar.personalised_covers.aws.region }}",
+	'aws_bucket' => "{{ pillar.elife.env}}-{{pillar.personalised_covers.aws.bucket}}",
+	'elife_api' => "http://{{ pillar.elife.env }}--{{ pillar.personalised_covers.api }}",
```

Comment:
`aws_bucket` and `elife_api` are too strict values, they should be provided by the `pillar/personalised_covers.sls` file.
Then I will include into builder-private (before merge) the correct values. That keeps the formula more reusable rather than tying it down to our convention.